src/installer.js

```diff
@@ -414,6 +415,10 @@ module.exports = (data, callback) => {
 
   let options
 
+  if (process.umask() !== 0o0022 && process.umask() !== 0o0002) {
+    console.warn(`\x1b[1mThe current umask, ${process.umask().toString(8)}, is not supported. You should use 0022 or 0002\x1b[0m`)
```

Comment:
I'd rather not have control characters in console messages. In Electron Forge we use [`colors`](https://www.npmjs.com/package/colors).

Reply:
I think it's better to use the control characters this time, I don't want to add another dependency to the project just for having the message in bold.
If you are really against leaving it the way it is, I rather just remove the characters and leave the message without any "color"

Reply:
I'd drop the bold then. (I had no idea it was bold.)